scripts/install-worker.sh

```diff
@@ -31,6 +31,79 @@ validate_env_set KUBERNETES_VERSION
 validate_env_set KUBERNETES_BUILD_DATE
 validate_env_set PULL_CNI_FROM_GITHUB
 
+################################################################################
+### EC2 IMDS v2 Helper Functions ###############################################
```

Comment:
it'd be nice to throw these into `/usr/bin/` or something, since they also show up in:
- the bootstrap script: https://github.com/awslabs/amazon-eks-ami/blob/master/files/bootstrap.sh#L216
- max pods calculator: https://github.com/awslabs/amazon-eks-ami/blob/master/files/max-pods-calculator.sh#L85